app/views/partials/share.ejs

```diff
@@ -2,11 +2,10 @@
 <!--  -->
 <% const twitterUrl = htmlLang === 'nl' ? 'onzecorona.nl' : baseUrl %>
 <!--  -->
-<% const encode = encodeURIComponent; const message = encode(__(`CALL TO ALL US
-CITIZENS: This is how we defeat the corona virus: together we make the invisible
+<% const encode = encodeURIComponent; const message = encode(__(`CALL TO ALL Nepali: This is how we defeat the corona virus: together we make the invisible
 virus visible.`) + " " + __(`{{ amount }} people already shared their health
 status.`, { amount }) + " " + __(`Fill out this form - it takes 1 minute - and
-help the United States gain control over COVID-19.`)) %>
+help Nepal gain control over COVID-19.`)) %>
```

Comment:
This should still be `help the United States gain control over COVID-19`, and then you just change the translation in `en-NE` (not the key of the object but the value).

Reply:
@fossecode so I am guessing I need to update this in my config.json ```"SUPPORTED_LOCALES": ["en-NE","ne"],```. is there any other change that I need to make after I change this? Do I have to run `deploy-prod.sh` after that?

Reply:
Yes correct, just change the config and then run `deploy-prod.sh`